issue_drafts/issue_5.md

```diff
@@ -0,0 +1,173 @@
+# Issue 5: Combining wildcards
+
+Pipeline steps:
+- Combine site-specific mean temperature files
+- Plot mean temperatures by day of year
+- Update rule all 
+- Add in third lake
+
+Concepts learned:
+- Calling a rule that uses wildcards
+- Calling such a rule for multiple wildcard values
```

Comment:
I'm not sure I'm making this connection between the content and what these concepts are. Should they be removed or am I missing the lines where this is introduced?
```suggestion
```
```suggestion
- Calling a rule that uses wildcards
- Calling such a rule for multiple wildcard values
```

Reply:
I agree - these don't belong here. I must have written them out before I thought through the issue. I think I'll add the split-apply-combine paradigm here instead.